great_expectations/expectations/row_conditions.py

```diff
@@ -56,6 +61,43 @@ class ConditionParserError(GreatExpectationsError):
     pass
 
 
+class RowConditionParserType(enum.Enum):
+    GE = "ge"  # GE intermediate language
+    SPARK = "spark"
+    SPARK_SQL = "spark_sql"
```

Comment:
@anthonyburdi Do we support both `SPARK` and `SPARK_SQL`, or is one of them a placeholder for the future?  Thanks.

Reply:
Currently `SPARK` is a placeholder for the future, it would refer to the `pyspark.sql.Column` type.

Reply:
@anthonyburdi If there is no comment about this, can we please add it?  Thanks!

Reply:
Added!